### PR TITLE
feat: Add measurements repositories to EH

### PR DESCRIPTION
**Pull request by BjarkeMeier**
# Description

This PR adds the following code:
image

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/electrical_heating/electrical_heating/infrastructure/measurements_bronze/column_names.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿class ElectricityMarketColumnNames:
+    end_datetime = "end_datetime"
+    metering_point_id = "metering_point_id"
+    metering_point_id = "metering_point_id"
+    metering_point_type = "metering_point_type"
+    metering_point_type = "metering_point_type"
```
Comment (JesperGomes) [inline source/electrical_heating/electrical_heating/infrastructure/measurements_bronze/column_names.py:6]: duplicate

File: source/electrical_heating/electrical_heating/infrastructure/measurements_bronze/column_names.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿class ElectricityMarketColumnNames:
+    end_datetime = "end_datetime"
+    metering_point_id = "metering_point_id"
+    metering_point_id = "metering_point_id"
```
Comment (JesperGomes) [inline source/electrical_heating/electrical_heating/infrastructure/measurements_bronze/column_names.py:4]: duplicate

File: source/electrical_heating/electrical_heating/infrastructure/measurements_bronze/column_names.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    metering_point_type = "metering_point_type"
+    metering_point_type = "metering_point_type"
+    observation_time = "observation_time"
+    orchestration_instance_id = "orchestration_instance_id"
+    orchestration_type = "orchestration_type"
+    points = "points"
+    position = "position"
+    product = "product"
+    quality = "quality"
+    quantity = "quantity"
```
Comment (JesperGomes) [inline source/electrical_heating/electrical_heating/infrastructure/measurements_bronze/column_names.py:14]: duplicate

File: source/electrical_heating/electrical_heating/infrastructure/measurements_bronze/column_names.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+﻿class ElectricityMarketColumnNames:
```
Comment (JesperGomes) [inline source/electrical_heating/electrical_heating/infrastructure/measurements_bronze/column_names.py]: why is it called ElectricityMarket?

Comment (BjarkeMeier) [reply source/electrical_heating/electrical_heating/infrastructure/measurements_bronze/column_names.py]: Copy-paste 🙈

**Review by JesperGomes — APPROVED** [review]

**Pull request merged**